### ISSUE TITLE
az logic workflow show error

### ISSUE THREAD

**Issue by andreyza-tr** [Azure/azure-cli#19230]
## Describe the bug

**Command Name**
`az logic workflow show
Extension Name: logic. Version: 0.1.4.`

**Errors:**
```
Unable to deserialize to object: type, KeyError: 'FlowAccessControlConfiguration'
Traceback (most recent call last):
Temp\pip-unpacked-wheel-k34ct6ah\msrest\serialization.py, ln 1430, in _deserialize
Temp\pip-unpacked-wheel-k34ct6ah\msrest\serialization.py, ln 1588, in deserialize_data
KeyError: 'FlowAccessControlConfiguration'

...

Temp\pip-unpacked-wheel-k34ct6ah\msrest\serialization.py, ln 1430, in _deserialize
Temp\pip-unpacked-wheel-k34ct6ah\msrest\serialization.py, ln 1588, in deserialize_data
msrest.exceptions.DeserializationError: Unable to deserialize to object: type, KeyError: 'FlowAccessControlConfiguration'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az logic workflow show --resource-group {} --name {}`

## Environment Summary
```
Windows-10-10.0.19041-SP0
Python 3.6.8
Installer: MSI

azure-cli 2.15.1 *

Extensions:
logic 0.1.4

```

**Comment by yonzhan** [comment]
route to service team

**Comment by jsntcy** [comment]
The fix is available in logic 0.1.5 (latest version), and you can upgrade extension to latest version using command “az extension update –name logic” to try the fix.
